run/sigterm-handler/main.go

```diff
@@ -75,5 +77,10 @@ func main() {
 // [END cloudrun_sigterm_handler]
 
 func handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "Hello World!\n")
+	if r.URL.Query().Get("terminate") != "" {
```

Comment:
Please add a note that this is for testing purposes

Reply:
Done.